Buildings/Controls/OBC/FDE/OperatingModeSimulation.mos

```diff
@@ -0,0 +1,29 @@
+// Script generated by Dymola Mon Jul 13 13:28:30 2020
+// Plot commands
```

Comment:
All the `.mos` file should be in the folder `modelica-buildings/Buildings/Resources/Scripts/Dymola/Controls/OBC/...`. Like if you'd want to run the class `Buildings.Controls.OBC.FDE.PackagedRTUs.Validation.BASControllerSimulation`, you can create a script file `modelica-buildings/Buildings/Resources/Scripts/Dymola/Controls/OBC/FDE/PackagedRTUs/Validation/BASControllerSimulation.mos`. 

Please see for example class `Buildings.Controls.OBC.ASHRAE.G36_PR1.AHUs.MultiZone.VAV.Validation.Controller` and the script file `modelica-buildings/Buildings/Resources/Scripts/Dymola/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/Validation/Controller.mos`. In the class `Buildings.Controls.OBC.ASHRAE.G36_PR1.AHUs.MultiZone.VAV.Validation.Controller`, there is section:
```
 __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/Validation/Controller.mos"
    "Simulate and plot"),
```